test/runtests.jl

```diff
@@ -3,16 +3,26 @@ using FileIO
 using BSON: BSON
 using Random
 using ReferenceTests
+using Logging
+
+ci = tryparse(Bool, get(ENV, "CI", "false")) === true && !isinteractive()
+@show ci
+
+# minimize messages when testing locally, all messages on CI
+test_logger = ci ? current_logger() : NullLogger()
```

Comment:
I am still not convinced of the wisdom of this.
What do you think @johnnychen94 ?

Reply:
Agreed, I feel it's okay to let the test of ReferenceTest be verbose by default and it won't hurt for users of it. After all, this is strictly an unrelated change and is better to be avoided.

Reply:
@oxinabox, I have restored the verbose option by default.